### PR TITLE
Add Mapumental retirement page

### PR DESCRIPTION
**Pull request by sagepe**
I've had a stab at doing a simple page for Mapumental. We might want to review the text, particularly the "Why have we closed this site?" section.

Screenshot 2020-04-02 at 16 02 50

### CODE REVIEW

**Comment by sagepe** [comment]
Noting that we can uncomment the link to GitHub once we've open-sourced it.

File: mapumental_page.html [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+        <p>It&rsquo;s been used by the <a href="https://www.mysociety.org/services/case-study-visualising-the-accessibility-of-welsh-schools-by-public-transport/">Welsh Government</a> to understand how accessible their schools are for families who don&rsquo;t own a car. They used our existing dataset of public transport, but the <a href="https://www.mysociety.org/services/case-study-mapping-fire-engine-response-times-for-the-fpa/">Fire Protection Association</a> provided their own data — fire engine response times — to show how quickly a blaze at any given postcode could be attended.</p>
+      
+        <h2>Why have we closed this site?</h2>
+
+        <p>Maputmental was one of several useful social projects conceived and run by mySociety, <a href="https://www.mysociety.org/projects/">many of which</a> continue to flourish. It just wasn't possible to give the necessary time and resources to all of them, so we've made the difficult decision to close Mapumental and focus on running our core sites.</p>
```
Comment (davea) [inline mapumental_page.html]: Typo on `Maputmental`

**Review by davea — CHANGES_REQUESTED** [review]
One tiny typo, otherwise 👍

File: mapumental_page.html (hunk 1 above)
Comment (sagepe) [reply mapumental_page.html]: Doh! Cheers, fixed in coming rebase.

**Pull request merged**